### ISSUE TITLE
Plugin doesn't use the commit in phid for non-differential builds

### ISSUE THREAD

**Issue by ChaitanyaPramod** [uber/phabricator-jenkins-plugin#135]

**Comment by sectioneight** [comment]
Yes, it currently assumes you are passing the SHA as another parameter and configuring your SCM plugin to pull the correct SHA. We need some documentation on how this is supposed to be configured, and if possible we should fetch commit metadata and reset to that.

**Comment by ChaitanyaPramod** [comment]
Whoa! didn't know that. Will configure SHA parameter.
Does the PHID contain the commit SHA1?

(Thanks for the ultra-quick response)

**Comment by sectioneight** [comment]
This is how I set up my test install to verify the functionality:

![image](https://cloud.githubusercontent.com/assets/97087/13365459/04ba7834-dc8a-11e5-993f-59a20c839909.png)

And then in jenkins, you add a string parameter for `SHA` and tell the git plugin to check out that sha
